src/FactoryMuffin.php

```diff
@@ -362,4 +371,20 @@ private function loadDirectory($path)
             require $file->getPathName();
         }
     }
+
+    /**
+     * Camelize string.
+     *
+     * Transforms a string to camel case (e.g. first_name -> firstName).
+     *
+     * @param string $str String in underscore format
+     *
+     * @return string $str translated into camel caps
```

Comment:
Just `@return string` please. The variable name makes no sense, and will be treated as part of the actual description, and our cs is that we don't include descriptions on return values. We put such descriptions in the method level description.